tests/test.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 if [ $# -eq 0 ]
 then
-  python ../venv/bin/nosetests --nocapture
+  python ../venv/bin/nosetests --nocapture -s
```

Comment:
Doesn't `--nocapture` and `-s` do the same thing?

Reply:
Oh whoops this was left over from me trying to get print statements to show.  Will remove b/c it does, in fact, do the same thing as `--nocapture`